dashboard/test/controllers/script_levels_controller_test.rb

```diff
@@ -414,6 +414,11 @@ def assert_caching_enabled(cache_control_header, max_age, proxy_max_age)
     assert_equal "/s/laurel/stage/2/puzzle/2", build_script_level_path(@custom_s2_l2)
   end
 
+  test "build_script_level_url" do
+    assert_equal "/s/laurel/stage/1/puzzle/1", build_script_level_path(@custom_s1_l1)
+    assert_equal "http://test.host/s/laurel/stage/1/puzzle/1", build_script_level_url(@custom_s1_l1)
+  end
```

Comment:
Looks like Laurel added it in this commit https://github.com/code-dot-org/code-dot-org/commit/6292d2b83d787a2437ba09fada1d00095f32507b.